src/fundus/publishers/de/__init__.py

```diff
@@ -158,7 +158,7 @@ class DE(PublisherEnum):
 
     NTV = PublisherSpec(
         name="N-Tv",
-        domain="https://www.ntv.de/",
+        domain="https://www.n-tv.de/",
```

Comment:
```suggestion
        domain="https://n-tv.de/",
```

Reply:
`https://www.n-tv.de/` is the correct domain.